bench/dune

```diff
@@ -0,0 +1,16 @@
+(executable
+ (name bench)
+ (public_name bench)
+ (package ocamlformat-bench)
+ (libraries bechamel bechamel-js ocamlformat_lib stdio))
+
+(rule
+ (alias runbench)
+ (action
+  (run ./bench.exe)))
+
+(rule
+ (alias runtest)
+ (package ocamlformat-bench)
+ (deps bench.exe)
+ (action (progn)))
```

Comment:
I think you can remove the action. That will just record `bench.exe` as a dependency to `@runtest`.

An alternative that I'm using sometimes is to check for `argv` in the executable, do nothing if there are no arguments, and run the bench otherwise. That way you can use a `(test)` stanza (running `dune runtest` will build the test but do nothing) and a `runbench` action that calls it with an argument.